packages/workspace-tools/src/paths.ts

```diff
@@ -60,6 +15,7 @@ export function findProjectRoot(cwd: string) {
   return workspaceRoot || findGitRoot(cwd);
 }
 
+/** @deprecated */
```

Comment:
This was included in the initial repo creation but seems to be unused in workspace-tools, beachball, backfill, and lage